frontend/apps/marketing/src/components/common/entries/LinkEntry.ts

```diff
@@ -0,0 +1,10 @@
+import {BaseEntry} from 'contentful';
+
+export type LinkEntry = BaseEntry & {
```

Comment:
This should be moved to `src/components/link` as it relates to the structure of that component.  See above comment about enhancing the type